include/spirv-tools/libspirv.h

```diff
@@ -246,7 +246,10 @@ typedef enum spv_number_kind_t {
 
 typedef enum spv_text_to_binary_options_t {
   SPV_TEXT_TO_BINARY_OPTION_NONE = SPV_BIT(0),
```

Comment:
Pre-existing problem, similar for disassembler options.  NONE should have expanded bitmask value of 0, not 1.  :-)
Don't fix it, it would be confusing.